src/js/blue-button/components/DownloadLink.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import classNames from 'classnames';
+
+import { apiRequest } from '../utils/helpers';
+
+class DownloadLink extends React.Component {
+  constructor(props) {
+    super(props);
+    this.downloadHealthRecord = this.downloadHealthRecord.bind(this);
+    this.downloadUrl = null;
+    this.state = { downloading: false };
+  }
+
+  downloadHealthRecord(e) {
+    e.preventDefault();
+
+    if (this.state.downloading) return;
+
+    if (this.downloadUrl) {
+      window.open(this.downloadUrl, '_blank');
+      return;
+    }
+
+    this.setState({ downloading: true });
+    const requestUrl = '/v0/health_records';
+    apiRequest(
```

Comment:
Should this request send the file type too (.txt/.pdf)? Not sure how the API handles that